### PR TITLE
Implementation for sending Requests to backend

### PR DESCRIPTION
**Pull request by ArjunM98**

### CODE REVIEW

File: app/src/main/java/com/veephealthoutloud/healthoutloud/ApplicationActivity.java
Comment (chouvinc) [inline app/src/main/java/com/veephealthoutloud/healthoutloud/ApplicationActivity.java]: typo

File: app/src/main/java/com/veephealthoutloud/healthoutloud/Classes/Requests.java
Comment (chouvinc) [inline app/src/main/java/com/veephealthoutloud/healthoutloud/Classes/Requests.java]: This looks like a typo but not sure since i'm not familiar w/ volley (the double \\)

**Review by chouvinc — CHANGES_REQUESTED** [review]
Looks good to me other than the 2 typos -- won't know for sure if it'll work until we run the code in an emulator. Fix the typos and I'll approve (if you want, otherwise we can wait until it works to merge it in).

File: .idea/modules.xml
Comment (JustCourt) [inline .idea/modules.xml:9]: newline

File: app/src/main/java/com/veephealthoutloud/healthoutloud/ApplicationActivity.java
Comment (JustCourt) [inline app/src/main/java/com/veephealthoutloud/healthoutloud/ApplicationActivity.java]: since this doesn't extend the Activity class, might be helpful to change the name to something more specific (without "Activity")

**Review by chouvinc — APPROVED** [review]

**Pull request merged**
